forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -1659,11 +1659,23 @@ private boolean stateBasedAction_Saga(Card c, CardCollection sacrificeList) {
 
     private boolean stateBasedAction_Battle(Card c, CardCollection removeList) {
         boolean checkAgain = false;
-        if (!c.getType().isBattle()) {
-            return false;
+        if (!c.isBattle()) {
+            return checkAgain;
+        }
+        if (((c.getProtectingPlayer() == null || !c.getProtectingPlayer().isInGame()) &&
+                (game.getCombat() == null || game.getCombat().getAttackersOf(c).isEmpty())) ||
```

Comment:
@tool4ever why the extra Combat check there?

Reply:
> 704.5w. If a battle has no player in the game designated as its protector and **no attacking creatures are currently attacking that battle**, that battle's controller chooses an appropriate player to be its protector